### PR TITLE
New config option: only admins can create repos

### PR DESCRIPTION
**Pull request by nounoursheureux**
Useful if you just use gitea to host your code and do not want other people to use create repositories.

### CODE REVIEW

File: conf/locale/locale_en-US.ini [hunk 1]
```diff
@@ -661,6 +661,7 @@ config.register_email_confirm = Require E-mail Confirmation
 config.disable_register = Disable Registration
 config.show_registration_button = Show Register Button
 config.require_sign_in_view = Require Sign In View
+config.admin_only_repos = Only Admins can create Repos
```
Comment (tboerger) [inline conf/locale/locale_en-US.ini]: Maybe I'm wrong but IMHO this string have to be in all locales, even if it is currently not translated.

Comment (nounoursheureux) [inline conf/locale/locale_en-US.ini]: Done :+1:

**Comment by tboerger** [comment]
I have not tested that but generally codewise it looks ok to me.

File: conf/app.ini [hunk 2]
```diff
@@ -85,6 +85,7 @@ ENABLE_REVERSE_PROXY_AUTHENTICATION = false
 ENABLE_REVERSE_PROXY_AUTO_REGISTRATION = false
 ; Do not check minimum key size with corresponding type
 DISABLE_MINIMUM_KEY_SIZE_CHECK = false
+ADMIN_ONLY_REPOS = false
```
Comment (tboerger) [inline conf/app.ini]: Maybe you can add a comment here that describes the option?

**Comment by nounoursheureux** [comment]
Done. I also squashed the commits

**Comment by lunny** [comment]
I'm sorry. I think we should consider this with access. We only change the access table to do this and not write so many codes.

**Comment by lunny** [comment]
I think we should have a whole idea about the access control.

**Pull request closed without merging**